dist/windows/strawberry.nsi.in

```diff
@@ -145,13 +145,30 @@ UninstPage custom un.LockedListPageShow
 !insertmacro MUI_UNPAGE_FINISH
 
 !insertmacro MUI_LANGUAGE "English" ;first language is the default language
+!insertmacro MUI_LANGUAGE "Italian"
```

Comment:
If we really going to add this, we should at least have MUI_LANGUAGE macros for all languages.

Reply:
@jonaski 

I suggest to test the new instalelr build with english and italian.

If it will works as expected decide you which additional language include after.

Reply:
Please build new installer and provide me the link to download it for final test.

Reply:
You need to build the installer and make sure it works before you create the PR in the first place.

Reply:
@jonaski 

Could you create a temporary build?
I didn't have resources and knowledge to make a build in the rightw way.

Reply:
In #1154 you described yourself as a NSIS power user. Creating a installer is basically just dragging the nsi file over in a the makensis window, it couldn't get easier. Maybe I'm misunderstanding what you mean by knowledge and resources here?
I got too many things to do in this project that are already delayed, and the project is lacking contributors, I can't be manually building setups for you to test because you can't take the time to do it yourself.

Reply:
As explained if you can provide me an archive with source files collect by NSIS to create the installer I can nake the test on the installer.
Sorry for the errors but sometimine to write manually and use of ( { ) } and very similar and be get confusion.
I beleieve that someone that try to help you to make an improvements it's better then nothing.